test/unittests/libethereum/Block.cpp

```diff
@@ -37,7 +37,7 @@ BOOST_FIXTURE_TEST_SUITE(FrontierBlockSuite, FrontierNoProofTestFixture)
 
 BOOST_AUTO_TEST_CASE(bStates)
 {
-	TestBlockChain testBlockchain(TestBlockChain::defaultGenesisBlock());
```

Comment:
No need to change this file, the tests are already in `FrontierNoProofTestFixture`.
Others are  in `ConstantinopleTransitionTestFixture` which also uses `NoProof`

Reply:
This controdicts your previous requirement of explicit constructor in test blockchain class

Reply:
My comment was about `explicit` keyword in constructor declaration